npm/private/npm_translate_lock.bzl

```diff
@@ -331,6 +339,18 @@ def _gen_npm_imports(lockfile, attr):
                     else:
                         link_packages[link_package].append(dep_package)
 
+        # check if this package should be hoisted via public_hoist_packages
+        public_hoist_packages = []
+        public_hoist_packages = attr.public_hoist_packages.get(name, [])[:]
+        public_hoist_packages.extend(attr.public_hoist_packages.get(friendly_name, []))
+        if unfriendly_name:
+            public_hoist_packages.extend(attr.patches.get(unfriendly_name, []))
+        for public_hoist_package in public_hoist_packages:
```

Comment:
nice short code change is a good sign that the abstractions will endure more feature sprawl :)

Reply:
🌮